### ISSUE TITLE
[aws-vault-v7.0.0-beta2] [sso-session] '' missing in config file

### ISSUE THREAD

**Issue by FernandoMiguel** [99designs/aws-vault#1148]
```sh
$ ~/Desktop/aws-vault-v7.0.0-beta2 --debug --prompt=osascript exec sre-dev --
2023/02/20 12:01:12 aws-vault v7.0.0-beta2
2023/02/20 12:01:12 Using prompt driver: osascript
2023/02/20 12:01:12 Loading config file /Users/fernando/.aws/config
2023/02/20 12:01:12 Parsing config file /Users/fernando/.aws/config
2023/02/20 12:01:12 [keyring] Considering backends: [keychain]
2023/02/20 12:01:12 [sso-session] '' missing in config file
2023/02/20 12:01:12 [sso-session] '' missing in config file
2023/02/20 12:01:12 [keyring] Querying keychain for service="aws-vault", keychain="aws-vault.keychain"
2023/02/20 12:01:12 [keyring] Found 0 results
aws-vault: error: exec: Error getting temporary credentials: profile sre-dev: credentials missing
```

```ini
[default]
region = us-east-1

[profile sre-dev]
sso_start_url = https://foobar.awsapps.com/start
sso_region = us-east-1
sso_account_id = 12345678900
sso_role_name = AdministratorAccess
region = us-east-1
```

**Comment by mtibben** [comment]
I noticed the same, caused by #1088

@dima-vm can you please address this?

**Comment by mtibben** [comment]
Fixed in ba79e47d6e85e280bca71015168330bae1543e6b
